docs/en/reference/query-builder.rst

```diff
@@ -97,10 +102,9 @@ to work with them, here is the same example 6 re-written using
        ->orderBy('u.name', 'ASC');
 
 ``QueryBuilder`` helper methods are considered the standard way to
-build DQL queries. Although it is supported, using string-based
-queries should be avoided.  You are greatly encouraged to use
-``$qb->expr()->*`` methods. Here is a converted example 8 to
-suggested standard way to build queries:
+use the ``QueryBuilder``. The ``$qb->expr()->*`` methods can help you
+build conditional expressions dynamically. Here is a converted example 8 to
```

Comment:
The examples aren't counted. Does it make sense to call it "converted example 8" and can we remove the number from the docs?

Reply:
I didn't want to change this part of the sentence as it wasn't in the scope of the PR, but yes, these example numbers are confusing, I don't even know where/how to find the so called "example 8". Maybe something that existed before and was removed at some point?

Reply:
Let's keep this for another PR in the future. :-)